src/components/Button.js

```diff
@@ -46,26 +58,45 @@ export const Button = ({ disabled, invert, notFullWidth, onPress, title }) => {
       titleStyle={[
         styles.titleStyle,
         invert && styles.titleStyleInvert,
-        needLandscapeStyle && styles.titleStyleLandscape
+        needLandscapeStyle && styles.titleStyleLandscape,
+        extraLarge && styles.extraLargeTitle,
+        large && styles.largeTitle,
+        medium && styles.mediumTitle,
+        small && styles.smallTitle
       ]}
       disabledStyle={styles.buttonStyleDisabled}
       disabledTitleStyle={styles.titleStyle}
       buttonStyle={[
         styles.buttonStyle,
         invert && styles.buttonStyleInvert,
-        isDelete && styles.rejectStyle
+        isDelete && styles.rejectStyle,
+        extraLarge && styles.extraLarge,
+        large && styles.large,
+        medium && styles.medium,
+        small && styles.small
       ]}
       containerStyle={[styles.containerStyle, needLandscapeStyle && styles.containerStyleLandscape]}
       ViewComponent={invert || isDelete || disabled ? undefined : DiagonalGradient}
       useForeground={!invert}
       accessibilityLabel={`${title} ${consts.a11yLabel.button}`}
       disabled={disabled}
+      icon={{
+        name: iconName,
+        type: IconSetName.toLowerCase(),
+        containerStyle: {},
```

Comment:
why do we need that empty object here?

Reply:
As standard, there is a margin that `react-native-elements` library adds for icon. I overwrote this style because it is too much for us.

Reply:
would have been an important note in the commit or as comment in the code.
couldn't we pass a margin that fits for us? or 0? this would make more sense in reading the code, as an empty object in my eyes? it would not need further information, as one sees a styles passed and knows what's going on.

Reply:
I'll add a style here. 👍 Even if it doesn't work, it will be more understandable.

Reply:
what do you mean with doesn't work?

Reply:
That's what I meant, I just wrote it to mask the containerStyle, so it doesn't make any difference whether it's empty or `{margin: 0}`.
I think this problem has been eliminated with the newly updated icon prop.

https://github.com/smart-village-solutions/smart-village-app-app/commit/5933f8013e0cfaaab2a97c661ecfa3b13eda8857